tools_for_todoist/models/event.py

```diff
@@ -181,8 +217,12 @@ def save(self):
             updated_fields['extendedProperties'] = self._extended_properties
         if updated_fields:
             self.google_calendar.update_event(self._id, updated_fields)
+    
+    @property
```

Comment:
let's keep these as helper functions unless it's actually a property with a setter

Reply:
I thought it's more pythonic to have properties instead of getters, but sure. This is the style of the project (: